crates/pathfinder/examples/feeder_gateway.rs

```diff
@@ -23,7 +23,7 @@ pub struct ContractAddresses {
 /// Serve feeder gateway REST endpoints required for pathfinder to sync.
 ///
 /// Usage:
-/// `cargo run --release -p pathfinder --example feeder_gateway ./testnet2.sqlite`
+/// `cargo run --release -p pathfinder --example feeder_gateway ./testnet.sqlite`
```

Comment:
We originally named it goerli since it ran on goerli. And then integration was added which also ran on goerli so whoops bad choice 😁 

```suggestion
/// `cargo run --release -p pathfinder --example feeder_gateway ./goerli.sqlite`
```